cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -676,7 +676,10 @@ def supercheq(
         fidelities = gss.serialization.deserialize(json_dict["fidelities"])
         return circuits, fidelities
 
-    def target_info(self, target: str,) -> Dict[str, Any]:
```

Comment:
the trailing comma after `str` here is why the formatter want this to be multiple lines; it should be fine with
```python
    def target_info(self, target: str) -> Dict[str, Any]:
```